assembly/__tests__/safe_u128.spec.as.ts

```diff
@@ -0,0 +1,302 @@
+import { u128 } from '../../assembly/integer/safe/u128';
+
+describe("Basic operations", () => {
+  it("should Add Two Numbers 1", (): void => {
```

Comment:
I guess `it("...", () => {` much cleaner. So will be great if you remove unnecessary `: void` declaration

Reply:
Ah yeah it's an old habit because the compiler used to complain without it.